extension/src/init.rs

```diff
@@ -239,7 +239,7 @@ fn append_embedding_column(job_name: &str, schema: &str, table: &str, col_type:
     )
 }
 
-pub fn get_column_datatype(schema: &str, table: &str, column: &str) -> Result<String> {
+pub fn get_column_datatype(table: &str, column: &str) -> Result<String> {
```

Comment:
i think here we still need the `schema` parameter so that we can determine the data type of the specified column. otherwise, if two tables with same name but different schema, we will get multiple values in the result.